meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -126,20 +127,25 @@ else
     inform "No need to flash bootloader to a specific device."
 fi
 
+# Mount internal device boot partition to write bootloader config file on it
```

Comment:
NIT: maybe change the comment to say it is mounted for adding the splash to it and then add another comment bellow when it does the cp to say that it copies the internal bootloader config file there?